<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thrDialog</class>
 <widget class="QDialog" name="thrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="logoutpb">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>40</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/4400629.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1531</width>
     <height>921</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Gestion des employe</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>0</y>
       <width>1641</width>
       <height>891</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
/* Style for QGroupBox */
QRadioButton{ color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}

/* Style for QLineEdit (input box) */
QLineEdit {
    background-color: #3498db; /* Blue color for input boxes */
    border: 1px solid #2980b9; /* Border color for input boxes */
    border-radius: 5px; /* Border radius for input boxes */
    padding: 5px; /* Padding for input boxes */
    color: black; /* Text color inside input boxes */
}

/* Style for QLabel (text label) */
QLabel {
    color: white; /* Text color for labels */
    font-size: 16px; /* Adjust font size as needed */
    font-weight: bold; /* Make the text bold if desired */
    margin: 5px; /* Adjust margin as needed */
}

/* Style for QPushButton */
QPushButton {
    background-color: #3498db; /* Blue color for buttons */
    border: 1px solid #2980b9; /* Border color for buttons */
    border-radius: 5px; /* Border radius for buttons */
    padding: 5px 10px; /* Adjust padding as needed */
    color: white; /* Text color on buttons */
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #2980b9; /* Darker blue on hover */
}

QPushButton:pressed {
    background-color: #21618c; /* Even darker blue when pressed */
}

/* Add more styles as needed for your specific application */
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>620</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>276</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Numbers only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>312</y>
        <width>58</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>312</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>Characters only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>276</y>
        <width>46</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>348</y>
        <width>65</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>poste</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="poste_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>348</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="placeholderText">
       <string>Characters only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>492</y>
        <width>76</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salaire_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>492</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Number only</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>420</y>
        <width>158</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>date d'embauche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="datee_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>420</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>YYYY-MM-DD</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_employes">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>180</y>
        <width>531</width>
        <height>591</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>384</y>
        <width>104</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Telephone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="telephone_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>384</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>+216</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>620</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>630</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>150</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>150</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sortAscButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>150</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>ASC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortDescButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>DESC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>100</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1201</width>
        <height>801</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/sports-nautiques.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-250</x>
        <y>-10</y>
        <width>1791</width>
        <height>921</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/img/health-still-life-with-copy-space.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>528</y>
        <width>102</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>528</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>564</y>
        <width>98</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pas_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>564</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pdf">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refrech_pb">
      <property name="geometry">
       <rect>
        <x>1250</x>
        <y>30</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Refrech</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>331</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Répartition par expérience de travail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Stats</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart_pie">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>140</y>
        <width>541</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chart_bar">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>200</y>
        <width>541</width>
        <height>701</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Yu Gothic UI Semibold</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Imprimer">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>240</y>
        <width>35</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>240</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_e">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>456</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>476</x>
        <y>456</y>
        <width>64</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>label_11</zorder>
     <zorder>lab_chart_pie</zorder>
     <zorder>lab_chart_bar</zorder>
     <zorder>label</zorder>
     <zorder>nom_e</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>poste_e</zorder>
     <zorder>label_9</zorder>
     <zorder>salaire_e</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>cin_e</zorder>
     <zorder>label_5</zorder>
     <zorder>datee_e</zorder>
     <zorder>tab_employes</zorder>
     <zorder>label_21</zorder>
     <zorder>telephone_e</zorder>
     <zorder>pb_modifier_2</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>radioButton</zorder>
     <zorder>label_7</zorder>
     <zorder>lineEdit</zorder>
     <zorder>label_8</zorder>
     <zorder>label_13</zorder>
     <zorder>user_e</zorder>
     <zorder>label_14</zorder>
     <zorder>pas_e</zorder>
     <zorder>pb_pdf</zorder>
     <zorder>refrech_pb</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_20</zorder>
     <zorder>Imprimer</zorder>
     <zorder>label_15</zorder>
     <zorder>id_e</zorder>
     <zorder>email_e</zorder>
     <zorder>label_4</zorder>
     <zorder>sortAscButton</zorder>
     <zorder>sortDescButton</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="backpb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>-20</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/LOG-removebg-preview.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1611</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/images.jfif);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>logoutpb</zorder>
  <zorder>tabWidget</zorder>
  <zorder>backpb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
